lib/matplotlib/collections.py

```diff
@@ -106,9 +93,71 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        Create a Collection
+        For all "plural" parameters, the type listed is the expected type when
+        passing a single value to be used for the whole collection. For any of
+        these inputs, a Sequence of the specified type is also allowed.
+
+        Parameters
+        ----------
+        edgecolors : color, default: :rc:`patch.edgecolor`
+            Edge color for each patch making up the collection. The special
+            value 'face' can be passed to make the edgecolor match the
+            facecolor.
+        facecolors : color, default: :rc:`patch.facecolor`
+            Face color for each patch making up the collection.
+        linewidths : float, default: :rc:`patch.linewidth`
+            Line width for each patch making up the collection.
+        linestyles : str or tuple, default: 'solid'
+            Valid strings are ['solid', 'dashed', 'dashdot', 'dotted', '-',
+            '--', '-.', ':']. Dash tuples should be of the form::
+
+                (offset, onoffseq),
 
-        %(Collection)s
+            where *onoffseq* is an even length tuple of on and off ink lengths
+            in points. For examples, see
+            :doc:`/gallery/lines_bars_and_markers/linestyles`.
+        capstyle : str, default: :rc:`patch.capstyle`
+            Style to use for capping lines for all paths in the collection.
+        joinstyle : str, default: :rc:`patch.joinstyle`
+            Style to use for joining lines for all paths in the collection.
+        antialiaseds : bool, default: :rc:`patch.antialiased`
+            Whether each pach in the collection should be drawn with
+            antialiasing.
+        offsets : (float, float), default: (0, 0)
+            A vector by which to translate each patch after rendering (default
+            is no translation). The translation is performed in screen (pixel)
+            coordinates (i.e. after the Artist's transform is applied).
+        transOffset : `.transforms.Transform`, default: IdentityTransform
+            A single transform which will be applied to each *offsets* vector
+            before it is used.
+        offset_position : {'screen' (default), 'data' (deprecated)}
+            If set to 'data' (deprecated), *offsets* will be treated as if it
+            is in data coordinates instead of in screen coordinates.
+        norm : `~.colors.Normalize`, optional, default: None
```

Comment:
```suggestion
        norm : `~.colors.Normalize`, default: None
```
A default automatically implies that this is optional. For brevity, we leave optional out in such cases. See https://matplotlib.org/devdocs/devel/documenting_mpl.html#default-values
> As opposed to the numpydoc guide, parameters need not be marked as optional if they have a simple default